README.md

```diff
@@ -1,12 +1,17 @@
-pkcs11-util
-===========
+pkcs11-util CI fork
+===================
 
 __pkcs11-util__ is a small tool to access cryptographic tokens through
 the PKCS#11 API.
 
+Upstream:
 [![Build Status](https://travis-ci.org/mbrossard/pkcs11.svg?branch=master)](https://travis-ci.org/mbrossard/pkcs11)
 [![Build status](https://ci.appveyor.com/api/projects/status/892m1yi8vmx9rmdh?svg=true)](https://ci.appveyor.com/project/mbrossard/pkcs11)
 
+Fork:
```

Comment:
This will need to be merged in a PR with the upstream, but one merge at a time for now.